### PR TITLE
helm chart

### PR DESCRIPTION
**Pull request by alv00**
helm chart

### CODE REVIEW

File: charts/ob-operator/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+appVersion: 1.16.0
+description: A Helm chart for OB-Operator
+name: ob-operator
+type: application
+version: 1.0.1
```
Comment (chris-sun-star) [inline charts/ob-operator/Chart.yaml]: set the version to 1.1.0, same as current release

File: charts/ob-operator/values.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Default values for ob-operator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image: oceanbasedev/ob-operator:1.2.0-snapshot
```
Comment (chris-sun-star) [inline charts/ob-operator/values.yaml]: use the current release oceanbase/ob-operator:1.1.0

File: index.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
```
Comment (chris-sun-star) [inline index.yaml]: no need to put index.yaml in master branch, it will be generated in gh-pages branch

**Review by chris-sun-star — CHANGES_REQUESTED** [review]

File: charts/ob-operator/templates/operator.yaml [hunk 4]
```diff
@@ -0,0 +1,448 @@
+apiVersion: v1
```
Comment (chris-sun-star) [inline charts/ob-operator/templates/operator.yaml:1]: apiVersion: v1
kind: Namespace
metadata:
  labels:
    control-plane: controller-manager
  name: {{ .Release.Namespace }}
---

add this and try `helm install ob xxx -n xxx` without --create-namespace

File: charts/ob-operator/Chart.yaml (hunk 1 above)
Comment (alv00) [reply charts/ob-operator/Chart.yaml]: done

File: charts/ob-operator/values.yaml (hunk 2 above)
Comment (alv00) [reply charts/ob-operator/values.yaml]: done

File: index.yaml (hunk 3 above)
Comment (alv00) [reply index.yaml]: done

File: charts/ob-operator/templates/operator.yaml (hunk 4 above)
Comment (alv00) [reply charts/ob-operator/templates/operator.yaml:1]: done

**Pull request merged**